server/server.go

```diff
@@ -175,7 +182,7 @@ func (server *Server) startHTTPServers() {
 	server.serverEntryPoints = server.buildEntryPoints(server.globalConfiguration)
 
 	for newServerEntryPointName, newServerEntryPoint := range server.serverEntryPoints {
-		serverMiddlewares := []negroni.Handler{server.accessLoggerMiddleware, server.loggerMiddleware, metrics}
+		serverMiddlewares := []negroni.Handler{server.accessLoggerMiddleware, metrics}
```

Comment:
I've updated the code such that the `server.accessLoggerMiddleware` `saveFrontend` and `saveBackend` are only used if access logging is configured.